### PR TITLE
changed npm run test to work on windows

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,6 +7,10 @@ branches:
     - master
     - dev
 
+install:
+  npm install -g mocha
+  npm install -g eslint
```
Comment (GianlucaGuarini) [inline .travis.yml]: I don't think this is necessary, they are already npm devDependencies

**Comment by GianlucaGuarini** [comment]
Honestly I would like to keep the `package.json` as it is because I mainly work in Unix environments. I am not concerned about windows at moment and I don't care about it either 😄

**Comment by ghost** [comment]
What about us? :'(

File: package.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
   "scripts": {
     "prepublish": "test $CI && exit 0 || npm run build && sed -i .bak 's/v\\([0-9]\\.[0-9]\\.[0-9]\\)/'`git tag | tail -n1`'/' README.md",
     "build": "rollup -c rollup.config.js > icaro.js",
-    "test": "npm run lint && ./node_modules/.bin/mocha test",
+    "test": "npm run lint && mocha test",
```
Comment (GianlucaGuarini) [inline package.json:11]: please do the same for `./node_modules/.bin/mocha`

Comment (ghost) [reply package.json:11]: @GianlucaGuarini what do i need to change there?

**Comment by GianlucaGuarini** [comment]
thank you

**Pull request merged**